.github/workflows/kotlin_ci.yml

```diff
@@ -32,5 +32,7 @@ jobs:
       run: ./gradlew build
     - name: Run ktlint
       run: ./gradlew ktlintCheck
-    - name: Run tests
+    - name: Run unit tests
       run: ./gradlew test
+    - name: Run integration tests
+      run: ./gradlew integrationTest -i
```

Comment:
Do we need the -i flag? Is the default output not enough?

Reply:
done, I removed it